test/admin-hpa/config/browserstack.yml

```diff
@@ -14,4 +14,5 @@ browser_caps:
     "os_version": "10"
     "browser": "Chrome"
     "resolution": "1024x768"
+    "browser_version": "81.0"
```

Comment:
Will this have to be updated as the latest comes out?

Reply:
Yep, I have now removed browser version all together, it will now pick up the latest version